mongo/options/createcollectionoptions.go

```diff
@@ -65,6 +79,20 @@ func (tso *TimeSeriesOptions) SetGranularity(granularity string) *TimeSeriesOpti
 	return tso
 }
 
+// SetBucketMaxSpan sets the value for BucketMaxSpan.
+func (tso *TimeSeriesOptions) SetBucketMaxSpan(dur time.Duration) *TimeSeriesOptions {
+	tso.BucketMaxSpan = &dur
+
+	return tso
+}
+
+// SetBucketRounding sets the value for BucketRounding.
+func (tso *TimeSeriesOptions) SetBucketRounding(dur time.Duration) *TimeSeriesOptions {
```

Comment:
@kevinAlbs @qingyang-hu I changed the setters to match the fields, from "SetBucketRoundingSeconds" to "SetBucketRounding".